File: src/java/src/main/java/com/example/ReadmeReader.java
package com.example;

import java.nio.file.*;

public class ReadmeReader {
    public static String readReadme(String filePath) throws Exception {
        Path path = Paths.get(filePath);
        if (!Files.exists(path)) {
            throw new Exception("README.md not found.");
        }
        return Files.readString(path);
    }
}

File: src/csharp/ReadmeReader/ReadmeReader.cs
using System;
using System.IO;

namespace ReadmeReaderApp
{
    public class ReadmeReader
    {
        public static string ReadReadme(string filePath)
        {
            if (!File.Exists(filePath))
            {
                throw new FileNotFoundException("README.md not found.");
            }
            return File.ReadAllText(filePath);
        }
    }
}
